src/dataflow/components/nodes/controls/relay-select-control.tsx

```diff
@@ -52,20 +52,26 @@ export class RelaySelectControl extends Rete.Control {
       const getChannelString = (ch?: NodeChannelInfo | "none") => {
         if (!ch && (!id || id === "none")) return kRelaySelectMessage;
         if (ch === "none") return "None Available";
-        if (!ch) return "Searching for " + id;
+        if (!ch) return "Finding " + id;
         let count = 0;
         channelsForType.forEach( c => { if (c.type === ch.type && ch.hubId === c.hubId) count++; } );
         return `${ch.hubName}:${ch.type}${ch.plug > 0 && count > 1 ? `(plug ${ch.plug})` : ""}`;
       };
-      const titleClass = getChannelString(selectedChannel).includes(kRelaySelectMessage) ? "label unselected" : "label";
       const options: any = [...channelsForType];
       if (!options.length) {
         options.push("none");
       }
+      const channelString = getChannelString(selectedChannel);
+      const titleClass = channelString.includes(kRelaySelectMessage)
+                         ? "label unselected"
+                         : "label";
+      const topItemClass = channelString.includes("Finding")
```

Comment:
This seems fragile -- relying on  the presence of a user-facing string that this very commit suggests is subject to change. Either it should be a constant like `kRelaySelectMessage`, or there should be some model property that drives both the channel string and the class(es).

Reply:
Fixed.